templates/credentials.epp

```diff
@@ -0,0 +1,6 @@
+[<%= $profile %>]
```

Comment:
The recommended way of using epp is to define all parameters with their types in the beginning